servers/src/main/java/tachyon/master/block/BlockMaster.java

```diff
@@ -345,8 +351,11 @@ public void commitBlock(long workerId, long usedBytesOnTier, String tierAlias, l
         if (masterBlockInfo == null) {
           masterBlockInfo = new MasterBlockInfo(blockId, length);
           mBlocks.put(blockId, masterBlockInfo);
-          writeJournalEntry(
-              new BlockInfoEntry(masterBlockInfo.getBlockId(), masterBlockInfo.getLength()));
+          BlockInfoEntry blockInfo = BlockInfoEntry.newBuilder()
+              .setBlockId(masterBlockInfo.getBlockId())
+              .setLength(masterBlockInfo.getLength())
+              .build();
+          writeJournalEntry(JournalEntry.newBuilder().setBlockInfo(blockInfo).build());
```

Comment:
Maybe this doesn't belong to this PR, but I am still quite bothered by the boilerplate code 
`writeJournalEntry(JournalEntry.newBuilder().setFoo(foo).build());`.

Can we overload method `writeEntry` and take parameters of different types, including `BlockInfoEntry`, `CompleteFileEntry` and etc? In that way, we still have the boilerplate code but in a centralized place, and probably less error-prune.